lib/make-badge.spec.js

```diff
@@ -23,6 +23,18 @@ describe('The badge generator', function () {
     makeBadge({ text: ['cached', 'not-cached'], format: 'svg' });
     assert.deepEqual([..._badgeKeyWidthCache.cache.keys()], ['cached']);
   });
+
+  it('should replace unknown json template with "default"', function () {
+    const jsonBadgeWithUnknownStyle = makeBadge({ text: ['name', 'Bob'], format: 'json', template: 'unknown_style' });
+    const jsonBadgeWithDefaultStyle = makeBadge({ text: ['name', 'Bob'], format: 'json', template: 'default' });
+    assert.strictEqual(jsonBadgeWithUnknownStyle, jsonBadgeWithDefaultStyle);
```

Comment:
Could you add a check that this value is a valid JSON string? And similarly check for svg below?

Reply:
What would you suggest to use for JSON validation? I've found https://stackoverflow.com/a/20392392/629762 and `isJSON` from https://github.com/chriso/validator.js.

Reply:
Or `expect(JSON.parse(jsonBadgeWithUnknownStyle)).to.deep.equal({name: "name", value: "Bob"})`

Reply:
Yea, that sounds great!

Reply:
Thanks! Done.